pyteal/ast/abi/util_test.py

```diff
@@ -593,3 +594,11 @@ def test_abi_type_translation(algosdk_abi, abi_string, pyteal_abi_ts, pyteal_abi
     )
     assert algosdk_abi == abi.algosdk_from_type_spec(pyteal_abi_ts)
     assert algosdk_abi == abi.algosdk_from_annotation(pyteal_abi)
+
+
+def test_sdk_abi_translation():
+    for case in ABI_TRANSLATION_TEST_CASES:
```

Comment:
```suggestion
@pytest.mark.parametrize("case", ABI_TRANSLATION_TEST_CASES)
def test_sdk_abi_translation(case):
```
indentation will be a bit off, but by separating out the tests this way, it will be easier to tell in CI which particular test case failes

Reply:
that's a good one ☝️